### DISCUSSION TITLE
HDR Output in Filament

### DISCUSSION THREAD

**Discussion by Nolram12345** [google/filament#7875] [Q&A] [answered]
With an increasing number of devices, both mobile, desktop and otherwise being equipped with or connected to displays capable of outputting in HDR, I was curious if it is possible to output HDR content properly with Filament? Has this been explored before?

**Comment by romainguy** [comment, marked as answer]
It's currently not available but we've prototyped it, yes.

**Reply by Nolram12345** [reply]
Awesome! Hoping to see this in Filament in the (hopefully not too distant) future.

**Reply by Nolram12345** [reply]
Wouldn't HDR output support in Filament primarily just require allowing the swapchain to be set to VK_FORMAT_A2B10G10R10_UNORM_PACK32, and allowing gamma correction to be skipped? Or would there be more involved than for a Filament implementation that which is blocking adoption of this feature?
Considering the quality of screens of all kinds these days - especially on mobile - I think an output feature like this could be very desirable.
